test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -424,9 +422,8 @@ export default class PerfCompare extends Component {
             }
         )
         let curAllVariantData = [];
-        let curVariantData, curMetricTable, curMatchingTestVariantIndex, curMatchingTestMetricIndex;
+        let curVariantData, curMetricTable, curMatchingTestVariantIndex, curMatchingTestMetricIndex, curBenchmarkVariant;
```

Comment:
`curBenchmarkVariant` isn't very reflective of the data it has. Maybe we should call it `benchmarkProps`.